### ISSUE TITLE
Rename this repository to "opensourcedesign.net"

### ISSUE THREAD

**Issue by jancborchardt** [opensourcedesign/opensourcedesign.github.io#172]
We should rename this repository to properly reflect the website name and make it more obvious for people where to contribute.

The problem we had was that when @simonv3 or I did that, the jobs and events weren't loading at all anymore on the main page. So probably it's referenced in some script or in the events/jobs repo.

**Comment by simonv3** [comment]
I thought it was a github thing?

**Comment by elioqoshi** [comment]
You can see all the places it's referenced here:
https://github.com/opensourcedesign/opensourcedesign.github.io/search?utf8=%E2%9C%93&q=opensourcedesign.github.io&type=

**Comment by elioqoshi** [comment]
I can do a PR replacing all those with opensourcedesign.net if you like

**Comment by simonv3** [comment]
Glancing at them I don't think changing any of those (though they should be changed) will affect the deployment process. They all look like they're part of documentation?

**Comment by jancborchardt** [comment]
Found the issue, I think. Somewhere it was looking for _events instead. Will check locally to fix it.

**Comment by jancborchardt** [comment]
Moving the events and jobs folder out into the root, with or without "_" makes it work strangely, but then other stuff is broken so not sure if this is a good way.

If it’s not fixable with something obvious, we can also simply roll it back by renaming the repo back to "opensourcedesign.github.io" and reverting https://github.com/opensourcedesign/opensourcedesign.net/commit/87e11857f8acb69cc73e7e5b70a5820588aa0e88

**Comment by belenbarrospena** [comment]
@jancborchardt should we consider renaming the repo and reverting that patch? I would offer to help, but I'd probably just make a mess of it :(

**Comment by jancborchardt** [comment]
@belenbarrospena yeah, renamed it back shortly after. Will look into combining the repos, but only in a pull request. :)

**Comment by AnXh3L0** [comment]
@jancborchardt I believe we can rename it back to opensourcedesign.net after we deploy the new version of the site?

**Comment by jancborchardt** [comment]
@AnXh3L0 that would be great! :) Yes, let’s switch, wait a bit for it to be settled, and then rename so we make sure it’s separate if there are any issues.

**Comment by AnXh3L0** [comment]
The repository has been renamed, marking as complete and closing issue.
